spec/javascripts/support/jasmine.yml

```diff
@@ -0,0 +1,124 @@
+# src_files
```

Comment:
That would be *awesome*, but it's not the end of the world if we add another line to .travis.yml.

In general, I'm +10000 to us adding JavaScript unit tests via some means, and I'm not too bothered about which framework we use.